packages/frontend/amp/components/TopMetaOpinion.tsx

```diff
@@ -38,6 +37,30 @@ const seriesStyle = (pillar: Pillar) => css`
     display: block;
 `;
 
+const bylineImageStyle = css`
+    display: block;
+    margin-right: -18px;
+    flex-shrink: 0;
+`;
+
+const bylineWrapper = css`
+    display: flex;
+    justify-content: space-between;
+
+    background-image: repeating-linear-gradient(
+        to bottom,
+        ${palette.neutral[86]},
+        ${palette.neutral[86]} 1px,
+        transparent 1px,
+        transparent 4px
+    );
+    background-repeat: repeat-x;
+    background-position: bottom;
+    background-size: 1px 29px;
```

Comment:
These *feel* a little like magic numbers to me, in that they're not immediately clear what they refer to. I think, in this context I would have a preference to name these values as variables immediately above the `css` block so that it's clearer what they refer to. i.e: do all the 1px values refer to the same design constraint, so if one changes, they all should? In which case, they should all refer to the same variable? Are any other numbers fractions of another? Variable dividing a variable. Does that make sense?

Reply:
Yes it makes sense definitely. And I agree.

We do this kind of thing all through AMP and also non-AMP because of the lack of design system/components library.

So I'm inclined to say we just leave it and let's discuss this in the giant code review. I am a bit sceptical of trying to give semantic names to these here though as it was really a case of pixel-pushing until it looked right.

Reply:
👍